{{cookiecutter.beat}}/LICENSE

```diff
@@ -0,0 +1,13 @@
+Copyright (c) 2016 {{cookiecutter.full_name}}
+
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
@cyrilleverrier Yeah, I was looking into this but came to the conclusion that it is probably too complex (not implementation, but making engineers pick a license during creation). People that know about licenses will replace it manually to fit their needs for all other cases I tried to pick the one which I think fits best for most beats. I was also tempted to leave out the LICENSE but I think as soon as something pushed on Github, it is important to have a license in. Let me know your thoughts.